Calculator/Calculator.Tests/Services/RpnServiceTests.cs

```diff
@@ -1,5 +1,88 @@
-﻿namespace Calculator.Tests.Services;
+﻿using Calculator.Services;
+
+namespace Calculator.Tests.Services;
 
 public class RpnServiceTests
 {
+    [Fact]
+    public void rpn_calculation_test1()
+    {
+        var service = new RpnService();
+        string input = "2 5 3 * +";
+        var result = service.СalculateRpn(input);
+
+        Assert.Equal(17,result);
+    }
+   
+     [Fact]
+    public void rpn_calculation_test2()
```

Comment:
All test names must be in the same style and look like *Subject*_*Action*_*ExpectedResult* (ex CalculateRpn_WithNegativeNumbersAndMixedOperations_ReturnsCorrectResult)